---a/net/minecraft/network/play/client/C13PacketPlayerAbilities.java
+++b/net/minecraft/network/play/client/C13PacketPlayerAbilities.java
@@ -1,12 +1,14 @@
 package net.minecraft.network.play.client;
 
 import java.io.IOException;
+
 import net.minecraft.entity.player.PlayerCapabilities;
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 
-public class C13PacketPlayerAbilities implements Packet
-{
+public class C13PacketPlayerAbilities implements Packet {
+
     private boolean invulnerable;
     private boolean flying;
     private boolean allowFlying;
@@ -17,9 +17,11 @@
     private float walkSpeed;
     private static final String __OBFID = "CL_00001364";
 
-    public C13PacketPlayerAbilities() {}
+    public C13PacketPlayerAbilities() {
+    }
 
-    public C13PacketPlayerAbilities(PlayerCapabilities capabilities)
-    {
+    public C13PacketPlayerAbilities(PlayerCapabilities capabilities) {
         this.setInvulnerable(capabilities.disableDamage);
         this.setFlying(capabilities.isFlying);
         this.setAllowFlying(capabilities.allowFlying);
@@ -29,8 +29,7 @@
         this.setWalkSpeed(capabilities.getWalkSpeed());
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         byte b0 = buf.readByte();
         this.setInvulnerable((b0 & 1) > 0);
         this.setFlying((b0 & 2) > 0);
@@ -40,27 +40,22 @@
         this.setWalkSpeed(buf.readFloat());
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         byte b0 = 0;
 
-        if (this.isInvulnerable())
-        {
-            b0 = (byte)(b0 | 1);
+        if (this.isInvulnerable()) {
+            b0 = (byte) (b0 | 1);
         }
 
-        if (this.isFlying())
-        {
-            b0 = (byte)(b0 | 2);
+        if (this.isFlying()) {
+            b0 = (byte) (b0 | 2);
         }
 
-        if (this.isAllowFlying())
-        {
-            b0 = (byte)(b0 | 4);
+        if (this.isAllowFlying()) {
+            b0 = (byte) (b0 | 4);
         }
 
-        if (this.isCreativeMode())
-        {
-            b0 = (byte)(b0 | 8);
+        if (this.isCreativeMode()) {
+            b0 = (byte) (b0 | 8);
         }
 
         buf.writeByte(b0);
@@ -69,64 +69,51 @@
         buf.writeFloat(this.walkSpeed);
     }
 
-    public void processPacket(INetHandlerPlayServer handler)
-    {
+    public void processPacket(INetHandlerPlayServer handler) {
         handler.processPlayerAbilities(this);
     }
 
-    public boolean isInvulnerable()
-    {
+    public boolean isInvulnerable() {
         return this.invulnerable;
     }
 
-    public void setInvulnerable(boolean isInvulnerable)
-    {
+    public void setInvulnerable(boolean isInvulnerable) {
         this.invulnerable = isInvulnerable;
     }
 
-    public boolean isFlying()
-    {
+    public boolean isFlying() {
         return this.flying;
     }
 
-    public void setFlying(boolean isFlying)
-    {
+    public void setFlying(boolean isFlying) {
         this.flying = isFlying;
     }
 
-    public boolean isAllowFlying()
-    {
+    public boolean isAllowFlying() {
         return this.allowFlying;
     }
 
-    public void setAllowFlying(boolean isAllowFlying)
-    {
+    public void setAllowFlying(boolean isAllowFlying) {
         this.allowFlying = isAllowFlying;
     }
 
-    public boolean isCreativeMode()
-    {
+    public boolean isCreativeMode() {
         return this.creativeMode;
     }
 
-    public void setCreativeMode(boolean isCreativeMode)
-    {
+    public void setCreativeMode(boolean isCreativeMode) {
         this.creativeMode = isCreativeMode;
     }
 
-    public void setFlySpeed(float flySpeedIn)
-    {
+    public void setFlySpeed(float flySpeedIn) {
         this.flySpeed = flySpeedIn;
     }
 
-    public void setWalkSpeed(float walkSpeedIn)
-    {
+    public void setWalkSpeed(float walkSpeedIn) {
         this.walkSpeed = walkSpeedIn;
     }
 
-    public void processPacket(INetHandler handler)
-    {
-        this.processPacket((INetHandlerPlayServer)handler);
+    public void processPacket(INetHandler handler) {
+        this.processPacket((INetHandlerPlayServer) handler);
     }
-}
+}
